packages/contracts-bedrock/justfile

```diff
@@ -1,6 +1,12 @@
 prebuild:
   ./scripts/checks/check-foundry-install.sh
 
+status:
```

Comment:
nit, would suggest renaming to `dep-status` or `status-deps` so it's more self-documenting